src/apps/components/AppListRow/AppListCardActions.tsx

```diff
@@ -109,7 +104,7 @@ const AppListCardActions: React.FC<AppListCardActionsProps> = ({
         removeInstall={removeInstallHandler}
       />
       {releaseDate && (
-        <Text color="default2" variant="body" size="small">
+        <Text color="default2" size={3}>
```

Comment:
suggestion: I think we should use `typeSize` or `size`, we shouldn't mix them.

Reply:
`typeSize` should be used - if there is `size` I probably made a mistake